settings/api/go.mod

```diff
@@ -0,0 +1,11 @@
+module api
+
+go 1.15
+
+require (
+	github.com/GDGVIT/socrates/schema v0.0.0-00010101000000-000000000000
```

Comment:
Why this weird version?

Reply:
The version signature was auto-generated by go